contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -33,7 +32,10 @@ interface ILinkAvailable {
 ///  this is a "trusless" upkeep, meaning it does not trust the caller of performUpkeep;
 /// we could save a fair amount of gas and re-write this upkeep for use with Automation v2.0+,
 /// which has significantly different trust assumptions
-contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AutomationCompatibleInterface {
+contract LinkAvailableBalanceMonitor is AccessControlEnumerable, AutomationCompatibleInterface {
+  bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
```

Comment:
Yes, we can use DEFAULT_ADMIN_ROLE with no problem and no change of this contract's behavior. Since we are also creating EXECUTOR_ROLE I just went ahead defining explicitly the two roles used on this contract, just because it's more intuitive.
I don't have a problem in just using the default one if you believe it's more readable/logical.